can/interfaces/slcan.py

```diff
@@ -50,7 +50,7 @@ class slcanBus(BusABC):
     LINE_TERMINATOR = b'\r'
 
     def __init__(self, channel, ttyBaudrate=115200, bitrate=None,
-                 sleep_after_open=_SLEEP_AFTER_SERIAL_OPEN,
+                 btr=None, sleep_after_open=_SLEEP_AFTER_SERIAL_OPEN,
                  rtscts=False, **kwargs):
         """
```

Comment:
> BTR register value to set custom can speed (overrides bitrate)

This is not true any more

```suggestion
         """
         :raise ValueError: if both *bitrate* and *btr* are set

```

Reply:
Thanks for your suggestion.

Reply:
Ahh, the

> (overrides bitrate)

is still there ^^

Reply:
Removed